pkg/app/instances/gce.go

```diff
@@ -38,6 +38,7 @@ type GCPIMConfig struct {
 	HostOrchestratorPort int
 	Network              string
 	Subnetwork           string
+	UsePrivateIP         bool
```

Comment:
let's default to private ip only, and add the config parameter to enabling external ip.